proposals/lip-0061.md

```diff
@@ -136,7 +148,7 @@ In this LIP, we will frequently use the following functions specified in LIP 005
 | `HASH_LENGTH` | 32 | Length in bytes of outputs of the SHA-256 hash function. |
 | `MAX_NUM_VALIDATORS` | 199 | The maximum number of validators that can be registered, see [LIP 0045](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0045.md#notation-and-constants). |
 | `MAX_LENGTH_AGGREGATION_BITS`|  `(MAX_NUM_VALIDATORS + 8 - 1) // 8` | The maximum length of `aggregationBits`.|
-| `MIN_CERTIFICATE_HEIGHT`| configurable per chain; for sidechains built with Lisk SDK, set to `genesisHeight +1` (defined in [LIP 0058][lip-0058#notation-and-functions]). For the mainchain in the Lisk Core v4 migration, set to be the first height for which all active validators are selected by PoS module based on their stake, (i.e., there is no validator selected due to being an initial validator specified in genesis data). | The smallest height for which a certificate can be produced.|
+| `MIN_CERTIFICATE_HEIGHT`| configurable per chain; for sidechains built with Lisk SDK, set to `genesisHeight + 1` where `genesisHeight` is the height of the genesis block. For the mainchain in the Lisk Core v4 migration, set to be the height of the last block before all active validators are selected by PoS module based on their stake (this block needs to be a block at the end of a round). | The smallest height for which a certificate can be produced.|
```

Comment:
```suggestion
| `MIN_CERTIFICATE_HEIGHT`| configurable per chain; for sidechains built with Lisk SDK, set to `genesisHeight + 1` where `genesisHeight` is the height of the genesis block. For the mainchain in the Lisk Core v4 migration, it is set to be the height of the last block before all active validators are selected by PoS module based on their stake (this block needs to be a block at the end of a round). | The smallest height for which a certificate can be produced.|
```

Reply:
Improved formulation. For brevity I do not use full sentences in the table though.